### PR TITLE
Remove "keep_pin" from the list of participants

### PR DESCRIPTION
**Pull request by tolzhabayev**
Remove "keep_pin" from the list of participants. This occurs when the list is generated while someone is presenting.

Here is a simple before and after comparison:
SCR-20230606-gac

### CODE REVIEW

**Review by geanfarias — APPROVED** [review]

**Comment by tolzhabayev** [comment]
@LambyPants any thoughts on if this can be merged?

**Comment by jacky-xbb** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online). Hurry, applications close on [March 1, 2024](https://jobsindeed.online).

## Job Description:
- **Position:** Senior Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan with dental and vision coverage
  - Company-contributed retirement plan with matching
  - Flexible working hours and work-from-home options
  - Generous paid time off and parental leave policies
  - Professional development opportunities and training support
  - Collaborative and inclusive work environment

## Requirements:
- Minimum of 5 years of experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Extensive experience with frameworks such as React, Angular, or Vue.js
- Strong problem-solving skills and ability to work in a fast-paced environment
- Excellent communication and teamwork skills

**Note:** This position is open to candidates from anywhere in the world. Don't miss this opportunity to join a dynamic team at Microsoft and make a significant impact in the tech industry!
BabakArtist, @buendiadas, @coreyshirk, @georgepadayatti, @yuwiggin, @pushpendrapratap, @Yhamskie30, @mfykmn, @Hudzen10, @Odement, @NoGeneral, @tuminfei, @jmiehau, @allenz682, @lambda-0x, @1Chr1s, @Mifrill, @rosellaa, @tkcolorado, @cl9200

**Comment by geanfarias** [comment]
@LambyPants 👀 
Do you have any predictions for the merge of this pr?

**Comment by LambyPants** [comment]
@geanfarias @tolzhabayev apologies for the (super) long wait. I am merging this and will try and do a deploy this weekend.

TL;DR life gets busy and I sorta forgot about this tool since I stopped using it myself. I am thrilled others are using it so will try and get this live asap. 

Thank you!

**Review by LambyPants — APPROVED** [review]

**Comment by geanfarias** [comment]
Thanks so much for the merge, @LambyPants. I confess that I was already starting to code a similar extension, even to test my skills.
Lamby, I'm currently receiving a notice from Google that your extension may be removed due to lack of privacy policy, do you think we can resolve this?
Furthermore, I am available to help maintain the code, review and merge the prs, if you are interested.
image

**Comment by LambyPants** [comment]
I just uploaded the new version, review normally takes ~24 hours

It should hopefully mix those warnings, too, but the manifest v2 -> v3 is fairly complex and I may have missed a detail or two. Will address if needed later this week.

**Comment by tolzhabayev** [comment]
@LambyPants I kind of already forgot about this PR a bit. We are now mostly just using the bookmarklet https://gist.github.com/tolzhabayev/8865374f7aa328f2a0d9ab16ffa37b18

**Comment by LambyPants** [comment]
Cool. @tolzhabayev fyi your change is now live - really appreciate your contribution and again apologies about the very long wait.
